src/Features/Core/Portable/UseThrowExpression/AbstractUseThrowExpressionDiagnosticAnalyzer.cs

```diff
@@ -77,15 +77,21 @@ private void AnalyzeOperation(OperationAnalysisContext context, INamedTypeSymbol
 
             var cancellationToken = context.CancellationToken;
 
-            var throwOperation = (IThrowStatement)context.Operation;
-            var throwStatement = throwOperation.Syntax;
+            var throwExpression = (IThrowExpression)context.Operation;
+            var throwStatementOperation = throwExpression.Parent;
+            if (throwStatementOperation.Kind != OperationKind.ExpressionStatement)
+            {
+                return;
+            }
+
+            var throwStatement = throwExpression.Syntax;
```

Comment:
It actually shouldn't matter, the IExpressionStatement and ThrowExpression use the same syntax node.